tests/units/Transvision/API.php

```diff
@@ -85,22 +85,22 @@ public function isValidRequestDP()
             ['http://foobar/api/wrong_version/tm/central/en-US/fr/Bookmark/', false],
             ['http://foobar/api/v1/wrong_service/central/en-US/fr/hello world', false],
 
-            // entity service
+            // Entity service
```

Comment:
I would keep all these lowercase, since they're actual names of services. Maybe `'entity' service` or `Service: entity` for clarity